x-pack/plugins/apm/public/components/app/service_groups/service_group_save/save_button.tsx

```diff
@@ -34,14 +35,62 @@ export function ServiceGroupSaveButton() {
 
   return (
     <>
-      <EuiButton
-        iconType={isGroupEditMode ? 'pencil' : 'plusInCircle'}
-        onClick={async () => {
-          setIsModalVisible((state) => !state);
-        }}
-      >
-        {isGroupEditMode ? EDIT_GROUP_LABEL : CREATE_GROUP_LABEL}
-      </EuiButton>
+      {isGroupEditMode && (
+        <ServiceGroupsTour
+          type="editGroup"
+          title={i18n.translate(
+            'xpack.apm.serviceGroups.tour.editGroups.title',
+            { defaultMessage: 'Edit this service group' }
+          )}
+          content={i18n.translate(
+            'xpack.apm.serviceGroups.tour.editGroups.content',
+            {
+              defaultMessage:
+                'Use the edit option to change the name, query, or details of this service group.',
+            }
+          )}
+        >
+          <EuiButton
+            iconType="pencil"
+            onClick={async () => {
+              setIsModalVisible((state) => !state);
+            }}
+          >
+            {i18n.translate('xpack.apm.serviceGroups.editGroupLabel', {
+              defaultMessage: 'Edit group',
+            })}
+          </EuiButton>
+        </ServiceGroupsTour>
+      )}
+
+      {!isGroupEditMode && (
+        <ServiceGroupsTour
+          type="createGroup"
+          title={i18n.translate(
+            'xpack.apm.serviceGroups.tour.createGroups.title',
+            { defaultMessage: 'Introducing service groups' }
+          )}
+          content={i18n.translate(
+            'xpack.apm.serviceGroups.tour.createGroups.content',
+            {
+              defaultMessage:
+                'Group services together to build curated inventory views that remove noise and simplify investigations across services. Groups are Kibana space-specific and available for any users with appropriate access.',
+            }
+          )}
+        >
+          <EuiButton
+            iconType="plusInCircle"
+            onClick={async () => {
+              setIsModalVisible((state) => !state);
+            }}
+          >
+            {i18n.translate('xpack.apm.serviceGroups.createGroupLabel', {
+              defaultMessage: 'Create group',
+            })}
+          </EuiButton>
+        </ServiceGroupsTour>
+      )}
+
```

Comment:
I wonder if at this point `<SaveButton>` should return `isGroupEditMode ? <EditButton> : <CreateButton>` defined in their own component files. I considered this pattern in the initial PR, but left as one component since it was so simple. What do you think?

Reply:
yeah, I think that will look way better. Thanks, I'll refactor it.

Reply:
done: https://github.com/elastic/kibana/pull/128068/commits/5aa07ce478dd4e3702611ede1abf9d1fc82d3e29